src/docs/product/accounts/sso/azure-sso.mdx

```diff
@@ -6,6 +6,12 @@ description: "Set up Azure Active Directory single sign-on (SSO) on Sentry."
 
 ## Installation
 
+<Alert level="warning">
+
+If you change your organization slug, you'll need to make the same update in the steps where you enter the SAML configuration values.
```

Comment:
@dashed - I was indicating that the change I suggested here should be applied in all the files.

Reply:
@imatwawana yep! I'm applying them everywhere 👍